tests/modules/seq2vec_encoders/boe_encoder_test.py

```diff
@@ -0,0 +1,64 @@
+# pylint: disable=no-self-use,invalid-name
+import numpy
+from numpy.testing import assert_almost_equal
+import torch
+from torch.autograd import Variable
+
+from allennlp.common import Params
+from allennlp.modules.seq2vec_encoders import BagOfEmbeddingsEncoder
+from allennlp.common.testing import AllenNlpTestCase
+
+class TestBagOfEmbeddingsEncoder(AllenNlpTestCase):
+    def test_get_dimension_is_correct(self):
+        encoder = BagOfEmbeddingsEncoder(embedding_dim=5)
+        assert encoder.get_input_dim() == 5
+        assert encoder.get_output_dim() == 5
+        encoder = BagOfEmbeddingsEncoder(embedding_dim=12)
+        assert encoder.get_input_dim() == 12
+        assert encoder.get_output_dim() == 12
+
+    def test_can_construct_from_params(self):
```

Comment:
Probably simpler to combine this test with the one above, but not a huge deal.  Just FYI for next time you do this, if you want to save yourself some work.